03_setup_pipelines/pipelines/example_workflow_A_plains3.yml

```
@@ -6,7 +6,7 @@ pipelines:
     compute-env: "$COMPUTE_ENV_PREFIX_plain_s3"       # Required, selects for Fusion CE created in earlier step
     profile: "$PIPELINE_PROFILE"                      # Optional, specify a test profile
     revision: "your_pipeline_revision"                # Required, specify a pipeline revision
-    config: "./nextflow.config"                       # Required, specify a config file with process resource labels for costs
+    config: "./plainS3.config"                       # Required, specify a config file with process resource labels for costs
```

```suggestion
    config: "./plainS3.config"                         # Required, specify a config file with process resource labels for costs
```
Comment alignment is triggering me